src/components/Nav/Nav.test.js

```diff
@@ -0,0 +1,11 @@
+import { screen, render } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import Nav from '.'
+
+describe('Nav', () => {
+    test('it renders a nav tag', () => {
+        render(<Nav />, { wrapper: MemoryRouter });
+        const nav = screen.queryByRole('navigation');
+        expect(nav).toBeInTheDocument();
+    })
+})
```

Comment:
Ditto what I said about the avatar test, and the same applies to all of your test suites.  :)